style.scss

```diff
@@ -0,0 +1,320 @@
+@font-face {
+  font-family: coco;
+  src: url('./media/fonts/cocogoose/cocogoose.woff');
+}
+
+$darkColor: #272a31;
+$brightColor: #d3d3d3;
+$primaryColor: #ec5242;
+$textColor: #4f4f4f;
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  font-family: lato, sans-serif;
+  font-size: 62.3%;
+}
```

Comment:
- [ ]For the UX (User experience) purpose, Let us increase the font-size of our page or body to look much more readable.

Reply:
Can you please suggest me **what should be the font size** because If I change the font size here in the body tag It'll be changed everywhere through out the website .

Reply:
## Optional

- When I remove `font-size: 62.5%;` from the body the app looks just fine try that and see